src/daft-core/src/series/array_impl/logical_array.rs

```diff
@@ -125,10 +125,6 @@ macro_rules! impl_series_like_for_logical_array {
                 self.0.str_value(idx)
             }
 
-            fn html_value(&self, idx: usize) -> String {
```

Comment:
how are we ensuring that we still have `html_value` if we drop this?

Reply:
We can't ensure that we have `html_value` on Series, instead we have to call the `html_value` function defined in daft-table. The issue is that the `ImageArray` definition is still in core, but the `html_value` implementation needed to be moved to the `daft-image` crate, so this logic needs to now happen further up in a crate that has access to both core and daft-image.